docs/ops.html

```diff
@@ -147,28 +147,31 @@
 
 <h4><a id="basic_ops_consumer_group" href="#basic_ops_consumer_group">Managing Consumer Groups</a></h4>
 
-With the ConsumerGroupCommand tool, we can list, delete, or describe consumer groups. For example, to list all consumer groups across all topics:
+With the ConsumerGroupCommand tool, we can list, describe, or delete consumer groups. Note that deletion is only available when the group metadata is stored in
+ZooKeeper. There's no need to delete it when using the <a href="http://kafka.apache.org/documentation.html#newconsumerapi">new consumer API</a> (where
+the broker handles coordination of partition handling and rebalance) as it is automatically deleted when the last member leaves.
+
+For example, to list all consumer groups across all topics:
 
 <pre>
- &gt; bin/kafka-consumer-groups.sh --zookeeper localhost:2181 --list
+ &gt; bin/kafka-consumer-groups.sh --bootstrap-server broker1:9092 --list
 
 test-consumer-group
 </pre>
 
 To view offsets as in the previous example with the ConsumerOffsetChecker, we "describe" the consumer group like this:
 
 <pre>
- &gt; bin/kafka-consumer-groups.sh --zookeeper localhost:2181 --describe --group test-consumer-group
+ &gt; bin/kafka-consumer-groups.sh --bootstrap-server broker1:9092 --describe --group test-consumer-group
 
 GROUP                          TOPIC                          PARTITION  CURRENT-OFFSET  LOG-END-OFFSET  LAG             OWNER
-test-consumer-group            test-foo                       0          1               3               2               test-consumer-group_postamac.local-1456198719410-29ccd54f-0
+test-consumer-group            test-foo                       0          1               3               2               consumer-1_/127.0.0.1
 </pre>
 
-
-When you're using the <a href="http://kafka.apache.org/documentation.html#newconsumerapi">new consumer API</a> where the broker handles coordination of partition handling and rebalance, you can manage the groups with the "--new-consumer" flags:
+If the group metadata is stored in ZooKeeper, pass <code>--zookeeper</code> instead of <code>bootstrap-server</code>:
```

Comment:
Do you think it's worth emphasizing somewhere that this applies to the old "high-level" consumer?